src/main/java/com/amaze/filemanager/fragments/FTPServerFragment.java

```diff
@@ -496,7 +498,11 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
      * @return address at which server is running
      */
     private String getFTPAddressString() {
-        return "ftp://" + FTPService.getLocalInetAddress(getContext()).getHostAddress() + ":" + FTPService.getPort();
+        String hostAddress = "";
+        InetAddress ia = FTPService.getLocalInetAddress(getContext());
+        if (ia != null)
+            hostAddress = ia.getHostAddress();
+        return "ftp://" + hostAddress  + ":" + FTPService.getPort();
```

Comment:
If `ia` is null `getFTPAddressString()` will return "ftp://:3333" (3333 being the port).
Why did you change this?